storage/innobase/log/log0chkp.cc

```diff
@@ -318,203 +318,10 @@ static void log_update_available_for_checkpoint_lsn(log_t &log) {
 
 /** @{ */
 
-<<<<<<< HEAD
-void log_files_header_fill(byte *buf, lsn_t start_lsn, const char *creator,
-                           bool no_logging, bool crash_unsafe) {
-  memset(buf, 0, OS_FILE_LOG_BLOCK_SIZE);
-
-  if (log_detected_format != UINT32_MAX) {
-    ut_a(log_detected_format <= LOG_HEADER_FORMAT_CURRENT);
-    mach_write_to_4(buf + LOG_HEADER_FORMAT, log_detected_format);
-  } else {
-    mach_write_to_4(buf + LOG_HEADER_FORMAT, LOG_HEADER_FORMAT_CURRENT);
-  }
-
-  mach_write_to_8(buf + LOG_HEADER_START_LSN, start_lsn);
-
-  /* set proper redo log format */
-  switch (log_detected_format) {
-    case LOG_HEADER_FORMAT_8_0_28:
-      strncpy(reinterpret_cast<char *>(buf) + LOG_HEADER_CREATOR,
-              LOG_HEADER_CREATOR_CURRENT,
-              LOG_HEADER_CREATOR_END - LOG_HEADER_CREATOR);
-      break;
-    case LOG_HEADER_FORMAT_8_0_19:
-      strncpy(reinterpret_cast<char *>(buf) + LOG_HEADER_CREATOR,
-              LOG_HEADER_CREATOR_8028,
-              LOG_HEADER_CREATOR_END - LOG_HEADER_CREATOR);
-      break;
-    case LOG_HEADER_FORMAT_8_0_3:
-      strncpy(reinterpret_cast<char *>(buf) + LOG_HEADER_CREATOR,
-              LOG_HEADER_CREATOR_8018,
-              LOG_HEADER_CREATOR_END - LOG_HEADER_CREATOR);
-      break;
-  }
-
-  ut_ad(LOG_HEADER_CREATOR_END - LOG_HEADER_CREATOR >= strlen(creator));
-
-  uint32_t header_flags = 0;
-
-  if (no_logging) {
-    LOG_HEADER_SET_FLAG(header_flags, LOG_HEADER_FLAG_NO_LOGGING);
-  }
-  if (crash_unsafe) {
-    LOG_HEADER_SET_FLAG(header_flags, LOG_HEADER_FLAG_CRASH_UNSAFE);
-  }
-  mach_write_to_4(buf + LOG_HEADER_FLAGS, header_flags);
-
-  log_block_set_checksum(buf, log_block_calc_checksum_crc32(buf));
-}
-
```

Comment:
we no longer need log_detected_format and this code to set appropriate header?

Reply:
not present in the latest commit.